conf/modules.config

```diff
@@ -544,7 +544,7 @@ process {
 
         if(params.updio) {
             withName: "^.*VCF_UPD_UPDIO:UPDIO\$" {
-                ext.prefix = {"updio_${meta.caller}"}
+                ext.prefix = {"updio_${meta.child}_${meta.caller}"}
```

Comment:
Why the child ID?

Reply:
Because UPDio is run for each child. If a family has multiple children the older ones would get overwritten.

Reply:
I updated this so one folder will be created per child (a bit similar to how automap outputs its results)